### ISSUE TITLE
[Gameobject] Tin node in the central Barrens floats off the ground.

### ISSUE THREAD

**Issue by therinrik** [dalaranwow/dalaran-wow#5314]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Tin vein in the Barrens directly southwest of the Lushwater Oasis floats out of reach off the ground.

**Expected behaviour**: Tell us what should happen instead
Vein should spawn on the ground.

**Steps to reproduce the problem**:

1. I found it while on a mining run. Issue may be random.
2. 
3. 

**Include proofs for this behaviour**
![wowscrnshot_022518_204933](https://user-images.githubusercontent.com/34615703/36651338-17e7e1c0-1a6e-11e8-9290-d90734269812.jpg)
![wowscrnshot_022518_205413](https://user-images.githubusercontent.com/34615703/36651339-17f765e6-1a6e-11e8-9c45-d6498531ca00.jpg)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
